pymc4/tests/test_random_variables.py

```diff
@@ -93,3 +93,20 @@ def test_rvs_logp_and_forward_sample(tf_session, randomvariable, kwargs):
         with pytest.raises(NotImplementedError) as err:
             dist.log_prob()
             assert "NotImplementedError: sample_n is not implemented: Binomial" == str(err)
+
+
+def test_rvs_backend_arithmetic():
```

Comment:
Not requiring a change but I think this would be more nicely tested in a parameterized test using lambda functions

Pros:
Each test would be split out by pytest and tested independently

Cons:
Its more code, less "direct", harder to read

Reply:
@canyon289 I thought about doing that, but the arithmetic is all implemented from the same backend class: I wasn't sure if it made sense to test the arithmetic for every random variable independently. I'm open to changing it, I'm just not sure what best test suite practices are.

Reply:
Ideally each test would test only one thing. In this case if the + operator fails it the test would stop and none of the other operators would be tested.

But it's really not too big of a deal. Feel free to merge if you feel like it :)

Reply:
Oh, I see! Should target for a follow up PR then, I suppose.

Reply:
I wouldn't worry about it. I think the time would be better spent elsewhere :)